docs/reference/release-notes/7.4.asciidoc

```diff
@@ -62,8 +64,10 @@ Infra/Scripting::
 * Drop stored scripts with the old style-id {pull}48078[#48078] (issue: {issue}47593[#47593])
 
 Machine Learning::
-* [ML] Fix detection of syslog-like timestamp in find_file_structure {pull}47970[#47970]
-* [ML] Reinstate ML daily maintenance actions {pull}47103[#47103] (issue: {issue}47003[#47003])
+* Fix detection of syslog-like timestamp in find_file_structure {pull}47970[#47970]
+* Reinstate ML daily maintenance actions {pull}47103[#47103] (issue: {issue}47003[#47003])
+* A reference to a temporary variable was causing forecast model restoration to 
+fail. The bug exhibited itself on MacOS builds with versions of clangd > 10.0.0. {ml-pull}688[#688]
```

Comment:
From an end user perspective this one should have been something more like:

* Fix possibility of crash when calculating forecasts that overflow to disk. {ml-pull}688[\#688]

The thing about clang 10+ refers to how it was reproducible in the unit tests.  The fix was for undefined behaviour so the bug could affect the production code differently.